src/SmartComponents/Compliance/ComplianceCard.js

```diff
@@ -114,9 +114,11 @@ const ComplianceCard = ({ fetchCompliance, complianceFetchStatus, complianceSumm
                                                         </SplitItem>
                                                         <SplitItem>
                                                             {intl.formatMessage(messages.compliantScore,
-                                                                { score: Math.trunc(
-                                                                    policy.attributes.compliant_host_count / policy.attributes.total_host_count
-                                                                ) }
+                                                                { score: +Math.trunc(
```

Comment:
`+` makes sure that the `.0` gets dropped, e.g if it's `100.0`, `50.0`, but not if it's any other number

Reply:
I'm confused why we're using `Math.trunc` and `toFixed(1)`. `trunc` always returns an integer with no decimal, and `toFixed(1)` always returns the number with 1 decimal again - you're guaranteed to get a `.0` on the end. Am I missing something?

```js
> (Math.trunc(5/10 * 100)).toFixed(1)
'50.0'
> (Math.trunc(5/11 * 100)).toFixed(1)
'45.0'
> (+Math.trunc(5/10 * 100)).toFixed(1)
'50.0'
> (+Math.trunc(5/11 * 100)).toFixed(1)
'45.0'
> (5/10 * 100).toFixed(1)
'50.0'
> (5/11 * 100).toFixed(1)
'45.5'
```

Reply:
Uh oh - removing it is the way to go indeed. 

```
+(5/11*100).toFixed(1)
45.5
+(5/10*100).toFixed(1)
50
```

Reply:
Ok, much better :) This `+` trick is a bit of a hack; this code converts a number to a string and then back to a number again, but I'm cool with it :stuck_out_tongue: 

```js
> 5/11*100
45.45454545454545
> (5/11*100).toFixed(1)
'45.5'
> +(5/11*100).toFixed(1)
45.5
```